packages/devui/src/Container/styles/index.ts

```diff
@@ -6,6 +6,8 @@ export const MainContainerWrapper = styled.div`
   height: 100%;
   width: 100%;
   flex-flow: column nowrap;
+  flex-shrink: 1;
+  min-height: 0;
```

Comment:
Can we use `overflow: auto` instead of `min-height: 0` here and below? Also if I understand the situation correctly, we don't need `flex-shrink: 1` since that's the default.